src/main/java/com/sogeti/SoSenseiPsyche/model/Game.java

```diff
@@ -47,11 +58,15 @@ public static String[] makeGuess(Scanner scanner) {
         String[] guess = scanner.nextLine().toUpperCase().split(" ");
 
         if (guess.length != CODE_LENGTH) {
+            System.out.println("Error: Your guess must contain exactly " + CODE_LENGTH + " colors.");
```

Comment:
Is this redundant now? Because in the feedback you also check for the length of the guess and give feedback